### ISSUE TITLE
casbin@3.0.1 is 18mb instead of the previous 2mb

### ISSUE THREAD

**Issue by rictorres** [casbin/node-casbin#78]
hi, there.

i've noticed that after https://github.com/casbin/node-casbin/commit/cab0ceb36f125e074314f37856f9543a9b2fc445 landed we now have [`jscodeshift`](https://www.npmjs.com/package/jscodeshift) as a dependency, the issue is that this module has 17mb.

we're heavy users of casbin inside lambdas however having a single function with 18mb+ is [extremely bad in terms of performance](https://github.com/casbin/node-casbin/pull/37).

must we use `jscodeshift`?

thanks in advance :)

[![image](https://user-images.githubusercontent.com/351568/60733608-a28da200-9f4d-11e9-9d59-ff88da858a55.png)](https://packagephobia.now.sh/result?p=casbin@3.0.1)

**Comment by hsluoyz** [comment]
@nodece

**Comment by hsluoyz** [comment]
Hi @rictorres , BTW, can you tell me where to get the image size as your picture? I want to add a badge in the repo's README to show the package size, as an alert :)

**Comment by rictorres** [comment]
@hsluoyz the image has a link to https://packagephobia.now.sh/result?p=casbin :)

badge:
```
[![install size](https://packagephobia.now.sh/badge?p=casbin)](https://packagephobia.now.sh/result?p=casbin)
```

[![install size](https://packagephobia.now.sh/badge?p=casbin)](https://packagephobia.now.sh/result?p=casbin)

**Comment by hsluoyz** [comment]
Thanks! Already added: https://github.com/casbin/node-casbin#node-casbin

We will fix this issue and publish a new release soon.

**Comment by rictorres** [comment]
awesome news! thanks, man :D

**Comment by hsluoyz** [comment]
Hi @rictorres , we have released v3.0.2:


![image](https://user-images.githubusercontent.com/3787410/60752677-218ce400-9ffb-11e9-9a06-b069fe71132e.png)

**Comment by rictorres** [comment]
beautiful! thanks again :D
